qaframework-bdd-tests/cypress/support/commands.js

```diff
@@ -76,14 +76,14 @@ Cypress.Commands.add('generateLabResults', (uuid) => {
     const labResults = {
         patient: uuid,
         encounterDatetime: "2012-02-05T04:03:02",
-        location: '6351fcf4-e311-4a19-90f9-35667d99a8af',
+        location: 'ba685651-ed3b-4e63-9b35-78893060758a',
```

Comment:
```suggestion
        location: DEFAULT_LOCATION_UUID,
```
Here also we can use the global variable